netZooPy/lioness/lioness.py

```diff
@@ -69,18 +77,11 @@ def __lioness_loop(self):
             idx = [x for x in range(self.n_conditions) if x != i]  # all samples except i
 
             with Timer("Computing coexpression network:"):
-                if self.computing=='gpu':
-                    import cupy as cp
-                    correlation_matrix = cp.corrcoef(self.expression_matrix[:, idx])
-                    if cp.isnan(correlation_matrix).any():
-                        cp.fill_diagonal(correlation_matrix, 1)
-                        correlation_matrix = cp.nan_to_num(correlation_matrix)
-                    correlation_matrix=cp.asnumpy(correlation_matrix)
-                else:
-                    correlation_matrix = np.corrcoef(self.expression_matrix[:, idx])
-                    if np.isnan(correlation_matrix).any():
-                        np.fill_diagonal(correlation_matrix, 1)
-                        correlation_matrix = np.nan_to_num(correlation_matrix)
+                subj_exp=self.expression_matrix[:, i]
+                correlation_matrix = self._normalize_network((((self.num_subj-1) * (self.correlation_matrix)) - (np.array([subj_exp]).T * subj_exp)) /(self.num_subj-2))
```

Comment:
great PR @dcolinmorgan . I'd like to suggest the following in this order:
- compute new correlation matrix
- fill diagonal with ones
- normalize matrix
and you can remove the lines 82->84

Reply:
right -- damn it normalize always beats me -- fill in diagonal THEN normalize